requirements.txt

```diff
@@ -0,0 +1 @@
+aiohttp>=3.3.0,<3.6.0
```

Comment:
I'm not sure if this is useful

Reply:
no requirements changes, thank

Reply:
please don't alter requirements unless you're adding a lib that's needed